cwltool/cwltest.py

```diff
@@ -92,7 +94,7 @@ def run_test(args, i, t):
         _logger.error(outstr)
         _logger.error("Parse error %s", str(e))
 
-    pwd = os.path.abspath(os.path.dirname(t["job"]))
+    os.path.abspath(os.path.dirname(t["job"]))
```

Comment:
That's interesting, I think this line and the comment block below it can go away entirely